packages/sdk/src/Base.ts

```diff
@@ -106,6 +108,28 @@ class Base {
     if (chainProviders) {
       this.chainProviders = chainProviders
     }
+
+    this.fees = config.bonderFeeBps[network]
+    this.destinationFeeGasPriceMultiplier = config.destinationFeeGasPriceMultiplier[network]
+
+    this.init()
+  }
+
+  async init () {
+    try {
+      const data = await this.getS3ConfigData()
+      if (data.bonders) {
+        this.bonders = data.bonders
+      }
+      if (data.bonderFeeBps) {
+        this.fees = data.bonderFeeBps
+      }
+      if (data.gasPriceMultiplier) {
+        this.destinationFeeGasPriceMultiplier = data.gasPriceMultiplier
```

Comment:
* I don't believe `this.destinationFeeGasPriceMultiplier` should be set to `gasPriceMultiplier`. Logically, if these are intended to accomplish different things, then setting them as equal is rather confusing. I may be missing something here though.
* I think this will happen unintentionally since `gasPriceMultiplier` defaults to `1`.

Reply:
I'll update the naming which is causing confusion

Reply:
The naming should be consistent now (https://github.com/hop-protocol/hop/pull/326/commits/79f100a3e97a6a2fb07c9b2dcab5a6ad405d85de)

`gasPriceMultiplier` is different than `destinationFeeGasPriceMultiplier`

- `destinationFeeGasPriceMultiplier` is what used be in `src/constants` as `GasPriceMultiplier` which was _only_ used for estimating destination gas cost fee if destination is L1, or ORU
- `gasPriceMultiplier` is the multiplier for `gasPrice` when sending transactions and this is used by developers